src/hooks/useConnectRelatedVenues.ts

```diff
@@ -170,15 +170,14 @@ export const useConnectRelatedVenues: ReactHook<
 
   const makeEventsQueryConfig = (
     doc: string,
-    storeAs: string
-  ): SparkleRFQConfig =>
-    ({
-      collection: "venues",
-      doc,
-      subcollections: [{ collection: "events" }],
-      orderBy: ["start_utc_seconds", "asc"],
-      storeAs,
-    } as SparkleRFQConfig); // @debt a little hacky, but we're consciously subverting our helper protections;
+    storeAs: ValidFirestoreKeys
```

Comment:
What was the thinking/motivation in changing this from `string` to `ValidFirestoreKeys`?

Reply:
it used to assign the return type of this function with `as`

![image](https://user-images.githubusercontent.com/63194656/104461209-b04acb80-55b7-11eb-89a5-785054733209.png)

I have refactored it to not use it. To do that, I needed to have `storeAs` as `ValidFirestoreKeys` type, which seems way more logical than having it as `string`

Reply:
Makes sense